tests/testthat/test_pairwise.R

```diff
@@ -163,6 +163,22 @@ test_that("test do_dist with cmd_scale", {
   expect_equal(result_kv[[4]], result_kv[[4]])
 })
 
+test_that("test do_dist with cmd_scale with normalize", {
+  loadNamespace("dplyr")
+  test_df <- data.frame(
+    row=rep(paste("row", seq(4)), each=6),
+    col=rep(paste("col", seq(6)), 4) ,
+    val=c(seq(7),seq(7),seq(10))
+  )
+  result_kv <- test_df %>%
+    do_dist(skv = c("row", "col", "val"), diag=TRUE, cmdscale_k = 3, normalize=TRUE)
+
+  expect_equal(ncol(result_kv), 4)
+  expect_equal(result_kv[[2]], result_kv[[2]])
```

Comment:
What is this testing? It should be equal all the time.

Reply:
Right.. Added expected values to the test.